src/acvp_aes.c

```diff
@@ -790,6 +791,12 @@ ACVP_RESULT acvp_aes_kat_handler(ACVP_CTX *ctx, JSON_Object *obj) {
                     rv = ACVP_INVALID_ARG;
                     goto err;
                 }
+            } else if (alg_id == ACVP_AES_GCM_SIV) {
+                if (ivlen != 96) {
```

Comment:
I guess we should make a #define for the GCM-SIV IV length

Reply:
And tag length - I will get on that